mina-archive/templates/deployment.yaml

```diff
@@ -69,6 +69,42 @@ spec:
 {{- $postgresHost := tpl .Values.archive.postgresHost . }}
 {{- $data := dict "postgresHost" $postgresHost "healthcheck" $.Values.healthcheck }}
 {{- include "healthcheck.archive.allChecks" $data | indent 10 }}
+        {{- if .Values.archive.initFromDump.enabled }}
+        - name: missing-blocks-guardian
```

Comment:
I am a little afraid about `name` collision if we were to create multiple archives in the same namespace. Maybe worth prepending `{{ .Release.Name }}`?

Reply:
This is a container name not a pod name. So it won't appear in `kubectl get pod`. For instance, the other container is called `archive`

Reply:
oh sorry, I missed that